guides/ageur-power-usecase.md

```diff
@@ -0,0 +1,63 @@
+# Get paid in agEUR, receive funds in your bank account
+
+![Untitled](/.gitbook/assets/angle-borrowmodulewp-announcement-light.png)
+
+Thanks to its powerful partners and integrations, agEUR can now be used to pay and create invoices, and agEUR can be easily transferred to your home bank account. All of this from almost anywhere in the world, on Ethereum or Polygon!
+
+This can be very useful on two sides: 1) for companies or DAOs wanting to pay their contributors in crypto, and 2) for contributors happy to be paid in crypto but who still need fiat in a bank account for their everyday lives. For example, some DAOs might not have a bank account, or just prefer to pay their contributors in crypto. However, these contributors usually need a proper proof of payment (invoice) and a way to transfer those funds to their bank account easily, without taking care of the exchange rate. This is all possible with agEUR today: anyone can start invoicing and getting paid in agEUR, and transfer them directly to EUR in their bank account!
```

Comment:
Can skip a line before the for example

Reply:
This is all possible with agEUR today: we should reformulate it in a way that makes it less "article"